lib/performance/performance.dart

```diff
@@ -397,7 +407,11 @@ tries['inclusiveFunctionTrie'] =
 */
 
 int clamp(int value, int min, int max) {
-  if (value < min) return min;
-  if (value > max) return max;
+  if (value < min) {
```

Comment:
nit: int now has a clamp method. in a separate CL perhaps switch to it instead.
https://api.dartlang.org/stable/2.0.0/dart-core/num/clamp.html

Reply:
ah, I was not aware; switched over to using this